### ISSUE TITLE
[Feature] Search in likes

### ISSUE THREAD

**Issue by ErikFontanel** [Soundnode/soundnode-app#509]
I've accumulated a very long list of likes and it's a real hassle to find tracks I've liked a few weeks/months ago. It'd be great if it's possible to do a keyword search through likes.

**Comment by matt-curtis** [comment]
Yeah. I've been using monthly playlists to combat this problem, but it still sucks. Also Soundcloud's Likes API seems to be very rigid in terms of how you get results (batched, you can't specify date, etc.)

**Comment by Pitros** [comment]
I've played with that for a while, and got searching using title or/and author. Would that be engouh? I also thought about date selecting (eg. older than [week, month, 3 months, 6 months, year]).

![image](https://cloud.githubusercontent.com/assets/6910670/12703825/a1aae8b0-c84c-11e5-8000-5d95bc041140.png)

@weblancaster what do you think about that kind of filtering for favs? Also resolves issue with duplicated tracks and infinite-scroll checks after each change for need to load more tracks (but it still may not load all tracks).
